src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,12 @@
 package mate.academy.service;
 
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null
```

Comment:
let's save `userService.findByEmail(email)` into a local variable and then reuse it

Reply:
I'm sorry i did not fix this one